### PR TITLE
FireEye HX: Bugfix - py2to3 minors

### PR DESCRIPTION
**Pull request by edro15**
#### Please check if your PR fulfills the following requirements:
- [X] Testing of all the changes has been performed (for bug fixes / features)
- [X] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [X] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [X] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [X] Verify all checks are passing.
- [X] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [X] Do NOT submit updates to dependencies unless it fixes an issue.
- [X] Limit your pull request to one application and submit multiple pull requests if necessary.
#### Please check the type of change your PR introduces:
- [ ] New App
- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Release Notes
- Fixed py2 to py3 bugs 
- Fixed minor issues in start_acquisition action

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+tox.ini
```
Comment (rgil-splunk) [inline .gitignore]: please don't add tox.ini to `.gitignore`: https://github.com/splunk-soar-connectors/lansweeper/pull/1#discussion_r783509354

**Review by rgil-splunk — APPROVED** [review]

**Review by ricardo-phantom — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (arathore-crest) [reply .gitignore]: Removed .gitignore

**Comment by mayurah** [comment]
👋🏼   @ricardo-phantom  when will this get merged?!

**Comment by arathore-crest** [comment]
Hi @mayurah,
Sanity testing is pending for this PR. Once it's complete, we'll merge it.

Thanks,
Aarav

**Pull request merged**